websites/K/KOLEO/presence.ts

```diff
@@ -0,0 +1,1003 @@
+				presenceData.smallImageText = "Przegląda zamówienie...";
+				presenceData.smallImageKey = Assets.Buy;
+			}
+		} else if (pathname.startsWith("/moje-konto")) {
+			if (document.querySelector(".woocommerce-form-login__submit")) {
+				presenceData.details = "Loguje się do swojego konta KOLEO.";
+				presenceData.smallImageText = "Loguje się...";
+				presenceData.smallImageKey = Assets.Writing;
+			} else {
+				presenceData.details = "Przegląda swoje konto KOLEO.";
+				presenceData.smallImageText = "Przegląda konto...";
+			}
+
+			if (pathname.startsWith("/moje-konto/lost-password")) {
+				presenceData.details = "Resetuje swoje hasło do konta KOLEO.";
+				presenceData.smallImageText = "Resetuje hasło...";
+				presenceData.smallImageKey = Assets.Writing;
+			}
+		} else if (pathname.startsWith("/regulamin")) {
+			presenceData.details = "Czyta regulamin sklepu KOLEO.";
+			presenceData.smallImageText = "Czyta regulamin...";
+			presenceData.smallImageKey = Assets.Reading;
+		} else if (pathname.startsWith("/polityka-prywatnosci")) {
+			presenceData.details = "Czyta politykę prywatności sklepu KOLEO.";
+			presenceData.smallImageText = "Czyta politykę...";
+			presenceData.smallImageKey = Assets.Reading;
+		} else if (pathname.startsWith("/francuska-11a")) {
+			presenceData.details = "Przegląda informacje kontaktowe KOLEO.";
+			presenceData.smallImageText = "Przegląda informacje...";
+		} else if (pathname.startsWith("/sklep-stacjonarny")) {
+			presenceData.details =
+				"Przegląda informacje o sklepie stacjonarnym KOLEO.";
+			presenceData.smallImageText = "Przegląda informacje...";
+		} else if (pathname.startsWith("/dostawa")) {
+			presenceData.details = "Przegląda informacje o dostawie w sklepie KOLEO.";
+			presenceData.smallImageText = "Przegląda informacje...";
+		} else if (pathname.startsWith("/interrail-najczesciej-zadawane-pytania")) {
+			presenceData.details =
+				"Przegląda najczęściej zadawane pytania o bilety Interrail.";
+			presenceData.smallImageText = "Przegląda pytania...";
+		} else {
+			const productTitle = document.querySelector(".product_title");
+			if (productTitle) {
+				presenceData.details = `Przegląda produkt${
+					privacySetting ? "" : ` - ${productTitle.textContent}`
+				}.`;
+				presenceData.smallImageText = "Przegląda produkt...";
+				if (!privacySetting) {
+					presenceData.state = `${
+						document.querySelector(".pwb-single-product-brands a")
+							? `Marka: ${
+									document.querySelector(".pwb-single-product-brands a")
+										.textContent
+							  } | `
+							: ""
+					}Cena: ${
+						document.querySelector(".price span.woocommerce-Price-amount bdi")
+							?.textContent
+					}`;
+					presenceData.buttons = [{ label: "Zobacz Produkt", url: href }];
+				} else {
+					presenceData.buttons = [
+						{ label: "Zobacz Katalog", url: "https://sklep.koleo.pl/katalog" },
+					];
+				}
+			} else NoPage(presenceData);
+		}
+	} else NoPage(presenceData);
```

Comment:
This is still s**t

Reply:
then i should just remove the function and put the code?

Reply:
I meant the whole presence.

Reply:
tf you on about bruh. 
it works? works. 
looks good? looks good.

Reply:
> tf you on about bruh. it works? works. looks good? looks good.

Works good, yes.
Looks good, no.

Reply:
then what would you change about the looks huh, it gives enough information on what the user is doing and if they got privacy mode on then it shows minial information that wont inpact there privacy

Reply:
> then what would you change about the looks huh, it gives enough information on what the user is doing and if they got privacy mode on then it shows minimal information that wont inpact there privacy

The code's look i meant. But my opinion isn't import, i'll leave it to the reviewers to form an opinion. So feel free to resolve this requested change.